plugins/jfrog-artifactory/src/components/JfrogArtifactoryRepository/tableHeading.tsx

```diff
@@ -4,19 +4,24 @@ import makeStyles from '@material-ui/core/styles/makeStyles';
 
 export const columns: TableColumn[] = [
   {
-    title: 'Version',
-    field: 'name',
+    title: 'Repos',
+    field: 'repos',
     type: 'string',
     highlight: true,
   },
   {
-    title: 'Repositories',
-    field: 'repositories',
+    title: 'Version Repository',
```

Comment:
Can you please explain me (I know nothing about Artifactory) how to read this field? How is this different from `field: repos` and from `field: name`?

Reply:
Hey! 👋 

Will get back on the other points as well (over time, when I get some spare time 😅 ) but this one is an interesting one. I am also not an "expert" by any means but I think this is possible:

```bash
- OurDockerImagesRepo
  |-> Development
  |   |--> CoolImage
  |
  |-> Releases
  |   |--> CoolImage
  |
  |-> Testing
      |--> CoolImage
```

This could be largely how we use Artifactory however, I want to be able to see where all versions of CoolImage are when it relates to an entity in Backstage.

Does that make sense? 😅